### ISSUE TITLE
Most used tags for user

### ISSUE THREAD

**Issue by alex-dp** [blazer82/analytodon#14]
if possible, stats should be presented on which tags a user uses most.

**Comment by blazer82** [comment]
Yes, thank you. I already got tag analytics on my roadmap - although I don't yet know when this will be available.

Good to know that this would be useful to you, though!
